tests/signal1.sh

```diff
@@ -0,0 +1,181 @@
+	assertEqualsMon "Validate pid calls" "1234" VALIDATEPID_CALL1_ARG1
+	assertEqualsMon "Validate signal calls" "1" VALIDATESIGNALNAME_CALLS
+	assertEqualsMon "Validate signal arg" "SIGINFO" VALIDATESIGNALNAME_CALL1_ARG1
+	assertEqualsMon "_is_pot_running calls" "1" ISPOTRUN_CALLS
+	assertEqualsMon "Send signal calls" "0" SENDSIGNAL_CALLS
+}
+
+test_pot_signal_020()
+{
+	pot-signal -p test-pot-run
+	assertEquals "Exit rc" "0" "$?"
+	assertEqualsMon "Error calls" "0" ERROR_CALLS
+	assertEqualsMon "Send signal calls" "1" SENDSIGNAL_CALLS
+	assertEqualsMon "Send signal arg1" "test-pot-run" SENDSIGNAL_CALL1_ARG1
+	assertEqualsMon "Send signal arg2" "SIGINFO" SENDSIGNAL_CALL1_ARG2
+	assertEqualsMon "Send signal arg3" "" SENDSIGNAL_CALL1_ARG3
+	assertEqualsMon "Send signal arg4" "" SENDSIGNAL_CALL1_ARG4
+	assertEqualsMon "Send signal arg5" "NO" SENDSIGNAL_CALL1_ARG5
+	assertEqualsMon "Send signal arg6" "NO" SENDSIGNAL_CALL1_ARG6
+
+	setUp
+	pot-signal -p test-pot-run -s SIGHUP
+	assertEquals "Exit rc" "0" "$?"
+	assertEqualsMon "Error calls" "0" ERROR_CALLS
+	assertEqualsMon "Send signal calls" "1" SENDSIGNAL_CALLS
+	assertEqualsMon "Send signal arg1" "test-pot-run" SENDSIGNAL_CALL1_ARG1
+	assertEqualsMon "Send signal arg2" "SIGHUP" SENDSIGNAL_CALL1_ARG2
+	assertEqualsMon "Send signal arg3" "" SENDSIGNAL_CALL1_ARG3
+	assertEqualsMon "Send signal arg4" "" SENDSIGNAL_CALL1_ARG4
+	assertEqualsMon "Send signal arg5" "NO" SENDSIGNAL_CALL1_ARG5
+	assertEqualsMon "Send signal arg6" "NO" SENDSIGNAL_CALL1_ARG6
+}
+
+test_pot_signal_021()
+{
+	pot-signal -p test-pot-run -s SIGHUP -P 1234 -f -C
+	assertEquals "Exit rc" "0" "$?"
+	assertEqualsMon "Error calls" "0" ERROR_CALLS
+	assertEqualsMon "Send signal calls" "1" SENDSIGNAL_CALLS
+	assertEqualsMon "Send signal arg1" "test-pot-run" SENDSIGNAL_CALL1_ARG1
+	assertEqualsMon "Send signal arg2" "SIGHUP" SENDSIGNAL_CALL1_ARG2
+	assertEqualsMon "Send signal arg3" "1234" SENDSIGNAL_CALL1_ARG3
+	assertEqualsMon "Send signal arg4" "" SENDSIGNAL_CALL1_ARG4
+	assertEqualsMon "Send signal arg5" "YES" SENDSIGNAL_CALL1_ARG5
+	assertEqualsMon "Send signal arg6" "YES" SENDSIGNAL_CALL1_ARG6
+}
+
+test_pot_signal_022()
+{
+	pot-signal -p test-pot-run -s SIGINFO -m grep -f -C
+	assertEquals "Exit rc" "0" "$?"
+	assertEqualsMon "Error calls" "0" ERROR_CALLS
+	assertEqualsMon "Send signal calls" "1" SENDSIGNAL_CALLS
+	assertEqualsMon "Send signal arg1" "test-pot-run" SENDSIGNAL_CALL1_ARG1
+	assertEqualsMon "Send signal arg2" "SIGINFO" SENDSIGNAL_CALL1_ARG2
+	assertEqualsMon "Send signal arg3" "" SENDSIGNAL_CALL1_ARG3
+	assertEqualsMon "Send signal arg4" "grep" SENDSIGNAL_CALL1_ARG4
+	assertEqualsMon "Send signal arg5" "YES" SENDSIGNAL_CALL1_ARG5
+	assertEqualsMon "Send signal arg6" "YES" SENDSIGNAL_CALL1_ARG6
+}
+
+setUp()
+{
+	common_setUp
+	ERROR_CALLS=0
+	HELP_CALLS=0
+	GETSIGNALNAMES_CALLS=0
+	VALIDATESIGNALNAME_CALLS=0
+	VALIDATESIGNALNAME_CALL1_ARG1=
+	VALIDATEPID_CALLS=0
+	VALIDATEPID_CALL1_ARG1=
+	SENDSIGNAL_CALLS=0
```

Comment:
All these lines shouldn't be necessary anymore, since common_setUp calls __mon_init, which in turn wipes all stored variables and CALLS checks default to 0.